apps/emqx_management/src/emqx_mgmt_api_publish.erl

```diff
@@ -102,12 +102,6 @@ fields(message) ->
                 required => false,
                 default => 0
             })},
-        {clientid,
```

Comment:
add `deprecated => true` instead of deleting it.

Reply:
deprecated fields will not be included in generated documents.
and the type check result will have this field excluded.

Reply:
Ok, that sound like a better way. I hope I have done it right this time.